client/post-editor/editor-status-label/index.jsx

```diff
@@ -125,7 +125,7 @@ export default React.createClass( {
 				} );
 				break;
 			case 'draft':
-				label = this.translate( '{{strong}}Draft saved{{/strong}} %(relativeTimeFromNow)s', {
+				label = this.translate( '{{strong}}Saved{{/strong}} %(relativeTimeFromNow)s', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( '{{strong}}Scheduled{{/strong}} %(relativeTimeFromNow)s' )` *ES Score: 19.25*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/edito-chrome-update)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).